pkg/network/ebpf/c/protocols/postgres/decoding.h

```diff
@@ -110,9 +110,10 @@ static int __always_inline skip_string(pktbuf_t pkt, int message_len) {
 // Main processing logic for the Postgres protocol. It reads the first message header and decides what to do based on the
 // message tag. If the message is a new query, it stores the query in the in-flight map. If the message is a command
 // complete, it enqueues the transaction and deletes it from the in-flight map. If the message is not a command complete,
-// it tries to read up to POSTGRES_MAX_MESSAGES messages, looking for a command complete message.
+// it tries to read up to POSTGRES_MAX_MESSAGES_PAR_TAIL_CALL*POSTGRES_MAX_TAIL_CALLS_FOR_MAX_MESSAGES messages, looking for a command complete message.
```

Comment:
is it true? postgres_entrypoint tries to read all message? or is it divided among multiple calls?